server/homebrew.api.js

```diff
@@ -43,9 +43,12 @@ const getBrew = (accessType)=>{
 				}
 			});
 		stub = stub?.toObject();
+		if(stub?.authors && !stub?.authors.includes(req.account.username)) {
+			throw 'Current logged in user does not have access to this brew.';
+		}
```

Comment:
Oh I went ahead and wrapped the initial authorship piece into this PR. This will prevent any unwanted edits.

Reply:
Actually I'm backing this out and putting in its own PR so we can release it separately if we want.